packages/plugin-flex/src/commands/flex/plugins/deploy.ts

```diff
@@ -123,6 +123,7 @@ export default class FlexPluginsDeploy extends FlexPlugin {
       args.push('--overwrite');
     }
 
+    await _verifyFlexUIConfiguration();
```

Comment:
What happens if you put this within a `progress` that has the third arg as `true` instead of false? Would be nice if we have a `Validating configuration` step or something

Reply:
setting disabled to `true` will also disable the spinner and progress title